tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,6 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    #raise Exception("Complete this test according to comments below.")
+    assert len(vendor.inventory) == 3
+    assert result == False
```

Comment:
Great consideration to check the inventory's length hasn't changed. We could also confirm there were no side affects that altered the individual contents of the inventory with :
```python
assert all(item in vendor.inventory for item in ["a", "b", "c"])
```
or by individually checking values:
```python
assert "a" in vendor.inventory
assert "b" in vendor.inventory
assert "c" in vendor.inventory
```